PresentScreenings/Entities/FilmInfo.cs

```diff
@@ -99,10 +106,10 @@ public override string ToString()
             var newLine = Environment.NewLine;
             var twoLines = newLine + newLine;
             var builder = new StringBuilder(Url + newLine);
-            if (FilmDescription != string.Empty)
+            if (RawFilmDescription != string.Empty)
             {
                 builder.AppendLine(newLine + "Description");
-                builder.AppendLine(FilmDescription);
+                builder.AppendLine(RawFilmDescription);
```

Comment:
Please consider to add a comment to explain why `RawFilmDescription` is used instead of `FilmDescription`.

Reply:
Done.